mkdocs_enumerate_headings_plugin/plugin.py

```diff
@@ -175,6 +178,9 @@ def on_post_page(self, output, page, config, **kwargs):
             )
             return output
 
+        if self.config.get("enumerate_nav", True):
+            output = output.replace(f"<h1>{page.chapter}. ", "<h1>")
+
```

Comment:
won't this remove all enumeration from titles? 

So the example gif from the README wouldn't have enumeration anymore?

<img width="327" alt="image" src="https://github.com/timvink/mkdocs-enumerate-headings-plugin/assets/5570380/fbf52846-039f-4b0a-b3b5-3b372059c621">